native/cocos/renderer/gfx-vulkan/VKSwapchain.cpp

```diff
@@ -309,8 +309,6 @@ bool CCVKSwapchain::checkSwapchainStatus(uint32_t width, uint32_t height) {
 
     destroySwapchain(gpuDevice);
 
-    gpuDevice->curBackBufferIndex = 0;
```

Comment:
While enter background, and then back to foreground. Some cases may add label node, it`ll trigger create texture buffers, and texture buffers bind to the index. After call doCreateSurface, texture buffers`s index maybe bind to the wrong index.

Reply:
The real cause of the crash is that the _buffersToBeUpdated cache instruction is used in the code, and the stored and used curBackBufferIndex may be modified